### PR TITLE
Server

### PR DESCRIPTION
**Pull request by saheedt**
**What does this PR do?**

- Sets up basic web server with route(s) entry

**Description of Task to be completed?**

- This task sets up a basic web server for handling http requests and routing

**How should this be manually tested?**

- None

**Any background context you want to provide?**

- None

**What are the relevant pivotal tracker stories?**

- #154811530

**Screenshots (if appropriate)**

- None

**Questions:**

- None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,28 +20,33 @@
...
-    "gulp-inject-modules": "^1.0.0",
-    "gulp-istanbul": "^1.1.3",
-    "gulp-jasmine-node": "^1.0.7",
-    "gulp-nodemon": "^2.2.1",
-    "nodemon": "^1.14.12"
+    "gulp-inject-modules": "1.0.0",
+    "gulp-istanbul": "1.1.3",
+    "gulp-jasmine-node": "1.0.7",
+    "gulp-nodemon": "2.2.1",
+    "nodemon": "1.14.12"
```
Comment (Tawakalt) [inline package.json]: I know I did this, but `nodemon` should be a dev dependency right?

Comment (saheedt) [reply package.json]: yes it should, should i make the change and re-push?

Comment (Tawakalt) [reply package.json]: Yes please.

Comment (saheedt) [reply package.json]: change made.

**Pull request merged**